propulsion-reactor/Program.fs

```diff
@@ -313,6 +313,11 @@ module Args =
         | [<AltCommandLine "-x">]           Port of int
         | [<AltCommandLine "-u">]           Username of string
         | [<AltCommandLine "-p">]           Password of string
+        | [<AltCommandLine "-Tp">]          ProjTcp
```

Comment:
maybe something else? convention is that `-T` represents a) a flag and b) something that can be stacked - so `Tp` makes both those aspects confusing (I don't have a better suggestion though, as it seems the adding of a `p` works well in general)

Reply:
As discussed keeping it -Tp